include/openssl/x509.h

```diff
@@ -637,6 +637,9 @@ int ASN1_item_digest(const ASN1_ITEM *it, const EVP_MD *type, void *data,
 
 int ASN1_item_verify(const ASN1_ITEM *it, X509_ALGOR *algor1,
                      ASN1_BIT_STRING *signature, void *data, EVP_PKEY *pkey);
+int ASN1_item_verify_ctx(const ASN1_ITEM *it, X509_ALGOR *algor1,
```

Comment:
There seems to be no documentation for this new function. Should it be public or internal?

Reply:
There is no documentation for the old either...

Reply:
I don't see this as a good enough reason to not document the new function. In most cases Travis would have actually complained about this missing documentation - but due to the issue fixed in #10980 it didn't. We should never again add new public symbols that aren't documented.

Reply:
I would like to document *all* those functions, not just one.  Will you be satisfied with a separate PR?

Reply:
Well...in principle "yes"....but....

https://github.com/openssl/openssl/pull/8442/files#r373407266

Reply:
Gotcha.  Still, it's a bit too much for this PR.  Make your mind up :wink:

Reply:
Ok...separate PR....but I'm going to check this one too ;-)

Reply:
I'll make them a priority after lunch, how's that sound?

Reply:
WFM